src/components/pages/Home/RenderHomePage.js

```diff
@@ -157,7 +157,31 @@ function RenderHomePage(props) {
     // savedCases.length,
     // savedJudges.length,
   ]);
-
+  const getPendingCases = () => {
```

Comment:
Note for the future:  we should move these axios calls into a separate folder or implement redux.